src/Contrib/Otlp/OtlpUtil.php

```diff
@@ -30,6 +37,16 @@ public static function method(string $signal): string
         return self::METHODS[$signal];
     }
 
+    public static function getHeaders(string $signal): array
+    {
+        $headers = Configuration::has(self::HEADER_VARS[$signal]) ?
+            Configuration::getMap(self::HEADER_VARS[$signal]) :
+            Configuration::getMap(Variables::OTEL_EXPORTER_OTLP_HEADERS);
+        $headers += self::getUserAgentHeader();
+
+        return array_map('urldecode', $headers);
```

Comment:
```suggestion
        return array_map('rawurldecode', $headers);
```

https://www.w3.org/TR/baggage/#value specifies RFC3986 encoding.